packages/ag-charts-community/src/chart/tooltip/tooltip.ts

```diff
@@ -136,12 +136,12 @@ const defaultTooltipCss = `
 }
 
 .${DEFAULT_TOOLTIP_CLASS}.${DEFAULT_TOOLTIP_DARK_CLASS} {
-    color: white;
-    background: #15181c;
+    // color: white;
```

Comment:
nit: technically this is invalid css, should be `/* color: white; */`

Reply:
Removed